quick-tabs/popup.js

```diff
@@ -357,7 +357,7 @@ function drawCurrentTabs() {
     // render only the tabs and closed tabs on initial load (hence the empty array [] for bookmarks)
     // also drop the first entry since that's the current tab =)
     renderTabs({
-      allTabs: bg.tabs.slice(1),
+      allTabs: bg.tabs.slice(1).sort(function(aTab, anotherTab) {return aTab.index - anotherTab.index}),
```

Comment:
this is great, but the issue is that this will result in a list of all of the 1st tabs from all windows, then the 2nd tabs from all windows, etc. sorting should be done on on tab.windowId first and then on tab.index (when the windowId  property is the same)

Reply:
fix:
```
      allTabs: bg.tabs.slice(1).sort(function(a, b) {
        return (a.windowId != b.windowId)? (a.windowId - b.windowId) : (a.index - b.index)
      }),
```

+ I agree that this alternative ordering should be an option.